simd/src/impl/Kokkos_AdvancedVectorExtensionsMacros.hpp

```diff
@@ -0,0 +1,34 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOS_ADVANCED_VECTOR_EXTENSIONS_MACROS_HPP
+#define KOKKOS_ADVANCED_VECTOR_EXTENSIONS_MACROS_HPP
+
+#include <Kokkos_Macros.hpp>
+
+#if defined(KOKKOS_ARCH_AVX) && !defined(__AVX__)
+#error "__AVX__ must be defined for KOKKOS_ARCH_AVX"
+#endif
+#cmakedefine KOKKOS_ARCH_AVX2
```

Comment:
```suggestion

```

Reply:
Fixed.

Reply:
I meant the empty line but anyway I think DongHun's suggestion is better.